change/@ni-nimble-components-79631013-9204-4a60-85e7-c3e288944a9f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm seeing odd behavior when moving the text cursor through a mention that's got bold and italic styling. The part of the mention before the cursor loses its styling while the part after it retains its styling. Perhaps this is expected behavior for our current implementation or isn't worth fixing, but it felt odd to me. It looks like this behavior existed before this change to some degree too.


https://github.com/ni/nimble/assets/10500124/7948a1f9-aefe-4e1c-9064-dee1702a83f8